src/routes/profile.js

```diff
@@ -52,19 +54,34 @@ export function profileRoutes() {
         typeof jwtData !== 'object' ||
         typeof jwtData.profileToken !== 'string'
       ) {
-        throw new ForbiddenError(`A profileToken was not found in the JWT.`);
+        if (ctx.state.jwtOriginalError) {
+          const error = ctx.state.jwtOriginalError;
+          // The JWT decoding middleware found an error earlier.
+          log.warn(
+            'delete.jwt.invalid',
```

Comment:
Note: I'll do a review of all the "codes" we use for logs, so that they're a bit more consistent accross the project. The goal will be to make it easier to extract important ones.